.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
Use a generated `.gitignore` 👉 https://www.gitignore.io/api/vapor,swift,linux,macos,windows,swiftpm

Reply:
Oh, thanks! Usually I use gitignore.io, but since `.gitignore` was autogenerated, I forgot to check it. 
Replaced ✅